protos/fpc/fpc.proto

```diff
@@ -0,0 +1,59 @@
+// Copyright IBM Corp. All Rights Reserved.
+// Copyright 2020 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+import "google/protobuf/any.proto";
+
+package fpc;
+option go_package = "github.com/hyperledger-labs/fabric-private-chaincode/internal/protos";
+
+message CC_Parameters {
+    // name of the chaincode
+    string chaincode_id = 1;
+
+    // version of chaincode, this is the expected mrenclave
+    // encoded as hexstring (as generated by common/enclave/generate_mrenclave.sh)
+    string version = 2;
+
+    // chaincode sequence number
+    int64 sequence = 3;
+
+    // name of channel
+    string channel_id = 4;
```

Comment:
Back to a comment of mine in #439 , here there is a mismatch with the fields. tlcc_mrenclave, channel_hash and/or mspid are now in attested_data, but... attested_data already contains cc_params. 
Either way, I don't think it changes much, it's only matter of consistency.

Reply:
mismatch resolved in #443